BUILDING.md

```diff
@@ -150,46 +150,44 @@ make STATIC=1
 
 # Via Docker container
 
-The following instructions assume you are at the parent directory of containerd source directory.
+> [!NOTE]
+> The following instructions assume you are at the **parent** directory of containerd source directory.
 
 ## Build containerd in a container
 
-You can build `containerd` via a Linux-based Docker container.
-You can build an image from this `Dockerfile`:
+You can build `containerd` via a Linux-based Docker container using the [Docker official `golang` image](https://hub.docker.com/_/golang/)
 
-```dockerfile
-FROM golang
-```
-
-Let's suppose that you built an image called `containerd/build`. From the
-containerd source root directory you can run the following command:
+From the **parent** directory of `containerd`'s cloned repo you can run the following command:
 
 ```sh
 docker run -it \
-    -v ${PWD}/containerd:/go/src/github.com/containerd/containerd \
-    -e GOPATH=/go \
-    -w /go/src/github.com/containerd/containerd containerd/build sh
+    -v ${PWD}/containerd:/src/containerd \
+    -w /src/containerd golang
 ```
 
-This mounts `containerd` repository
-
-You are now ready to [build](#build-containerd):
+This mounts the `containerd` repository inside the image at `/src/containerd` and, by default, runs a shell at that directory.
 
-```sh
-make && make install
-```
+Now, you are now ready to follow the [build instructions](#build-containerd):
 
 ## Build containerd and runc in a container
 
 To have complete core container runtime, you will need both `containerd` and `runc`. It is possible to build both of these via Docker container.
 
-You can use `git` to checkout `runc`:
+You can clone `runc` in the same parent directory where you cloned `containerd` and you should clone [the latest stable version of `runc`](https://github.com/opencontainers/runc/releases), e.g. v1.1.13:
 
 ```sh
-git clone https://github.com/opencontainers/runc
+git clone --branch v1.1.13 https://github.com/opencontainers/runc
```

Comment:
It should be noted that "v1.1.13" here should be replaced with the actual latest release.